services/docsrs/docsrs.service.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const Joi = require('joi')
+const { BaseJsonService } = require('..')
+
+const schema = Joi.array()
+  .items(
+    Joi.object({
+      build_status: Joi.boolean().required(),
+    })
+  )
+  .min(1)
+  .required()
+
+module.exports = class DocsRs extends BaseJsonService {
+  static category = 'build'
+  static route = { base: 'docsrs', pattern: ':crate/:version?' }
+  static examples = [
+    {
+      title: 'docs.rs',
+      namedParams: { crate: 'regex', version: 'latest' },
+      staticPreview: this.render({ version: 'latest', buildStatus: true }),
+      keywords: ['rust'],
+    },
+  ]
+
+  static defaultBadgeData = {}
```

Comment:
Let's keep the default value in here, as that will still be used in error scenarios (like not found errors on unknown crates/crate versions)

```suggestion
  static defaultBadgeData = { label: 'docs' }
```